src/docfx/build/toc/TableOfContentsMap.cs

```diff
@@ -156,8 +156,15 @@ private static int CompareTocCandidate(TocCandidate candidateX, TocCandidate can
             }
 
             var levenshteinDistanceCompareResult = candidateX.LevenshteinDistance - candidateY.LevenshteinDistance;
-            return levenshteinDistanceCompareResult == 0 ? StringComparer.OrdinalIgnoreCase.Compare(candidateX.Toc.SitePath, candidateY.Toc.SitePath)
-                : levenshteinDistanceCompareResult;
+            if (levenshteinDistanceCompareResult != 0)
+            {
+                return levenshteinDistanceCompareResult;
+            }
+            if (!StringComparer.OrdinalIgnoreCase.Equals(candidateX.Toc.SitePath, candidateY.Toc.SitePath))
+            {
+                StringComparer.OrdinalIgnoreCase.Compare(candidateX.Toc.SitePath, candidateY.Toc.SitePath);
```

Comment:
return here? Maybe prepare the result of `StringComparer.OrdinalIgnoreCase.Compare(candidateX.Toc.SitePath, candidateY.Toc.SitePath)`, if not 0, return it.